src/main/java/com/hillel/items_exchange/config/SwaggerConfig.java

```diff
@@ -20,6 +26,28 @@ public Docket api() {
                 .select()
                 .apis(RequestHandlerSelectors.any())
                 .paths(PathSelectors.any())
+                .build()
+                .securitySchemes(List.of(apiKey()))
+                .securityContexts(List.of(securityContext()));
+    }
+
+    private ApiKey apiKey() {
+        return new ApiKey("JWT", "Authorization", "header");
+    }
+
+    private SecurityContext securityContext() {
+        return SecurityContext.builder()
+                .securityReferences(defaultAuth())
+                .forPaths(PathSelectors.any())
                 .build();
     }
+
+    List<SecurityReference> defaultAuth() {
+        AuthorizationScope authorizationScope
+                = new AuthorizationScope("global", "accessEverything");
+        AuthorizationScope[] authorizationScopes = new AuthorizationScope[1];
+        authorizationScopes[0] = authorizationScope;
+
+        return List.of(new SecurityReference("JWT", authorizationScopes));
```

Comment:
You could do it in one line:
`return List.of(new SecurityReference("JWT", new AuthorizationScope[]{new AuthorizationScope("global", "accessEverything")}));`

Reply:
Done in commit "caaf656"